programs/vote/src/vote_instruction.rs

```diff
@@ -39,6 +39,9 @@ pub enum VoteError {
 
     #[error("authorized voter has already been changed this epoch")]
     TooSoonToReauthorize,
+
+    #[error("cannot set same authorized voter as latest authorized voter")]
+    ReauthorizedSameVoter,
```

Comment:
Should this be an error or a noop?

Reply:
Neither, I got rid of the error and now the behavior locks in the authorized voter again for that future epoch, preventing a switch